QuantExt/test/analyticlgmswaptionengine.cpp

```diff
@@ -35,7 +35,7 @@
 #include <ql/time/calendars/nullcalendar.hpp>
 #include <ql/time/calendars/target.hpp>
 
-#include <test-suite/utilities.hpp>
+#include <test/utilities.hpp>
```

Comment:
The headers in the `test-suite` directory are not installed, so I had to create a new one and copy into it the couple of macros that you're using in your tests.